cmd/apps/crossplane_app.go

```diff
@@ -0,0 +1,148 @@
+package apps
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/alexellis/k3sup/pkg"
+	"github.com/alexellis/k3sup/pkg/config"
+	"github.com/alexellis/k3sup/pkg/env"
+	"github.com/alexellis/k3sup/pkg/helm"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallCrossplane() *cobra.Command {
+	var crossplane = &cobra.Command{
+		Use:          "crossplane",
+		Short:        "Install crossplane",
+		Long:         `Install crossplane`,
```

Comment:
Maybe you could say what crossplane does here in the long? Like the headline. Take note of wrapping to 80 chars.